### DISCUSSION TITLE
Stop create Shortcut "C:\Users\Public\Desktop\SumatraPDF.lnk" on install application

### DISCUSSION THREAD

**Discussion by GowaP** [sumatrapdfreader/sumatrapdf#3958] [Q&A]
[FR:] Quel paramètre doit être spécifié sur la ligne de commande pour que lors de l’installation de SumatraPDF, le programme d’installation ne crée pas de raccourci sur le bureau ?

[EN:] Which parameter should be specified on the command line so that when installing SumatraPDF, the setup does not create a shortcut on the desktop?

Stop this action :
CreateAppShortcut(csidl=25), path=C:\Users\Public\Desktop\SumatraPDF.lnk

**Comment by GitHubRulesOK** [comment]
You can see all the switches at https://www.sumatrapdfreader.org/docs/Installer-cmd-line-arguments

Thus to not let the installer run its sequence with system generation of shortcuts and other system functions your only option would be modify the source and sign it or worse extract and install all the assignments in your own script (not recommended).

By far the easiest is run the installer then remove the unwanted function say by del "C:\Users\Public\Desktop\SumatraPDF.lnk" or similar depending where you installed it. However I dont see why you would need to as its perfect there for drag and drop to open any file.

**Reply by GowaP** [reply]
[FR:] Il serait plus simple que l'EXE d'installation soit standardisé pour avoir les paramètres suivants :
- ne pas créer de raccourci sur le bureau
- indiquer le groupe du menu Windows qui va recevoir les raccourcis du programme

[EN:] It would be simpler if the installation EXE was standardized to have the following parameters:
- Don't create shortcuts on the desktop
- Specify the Windows menu group that will receive the program's shortcuts
